### DISCUSSION TITLE
Enabling Library\options\Sort by Artist > year inhibits library page from being displayed.

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1765] [Fixed Issues]
I applied image rAudio-64bit-20240818.img and found that enabling the Library\options\Sort by Artist > year option inhibits the library page from being displayed.

![image](https://github.com/user-attachments/assets/00ab6cd1-5a57-42c3-ad36-52f2b5bac001)

This is a programmatic issue as the problem disappears simply be disabling the Sort by Artist > year option.

Also reference bug reports #1746 and  #1748

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Good find. 
Thanks.
Please try: 
```sh
sed -i '/D.albumyear/ d' /srv/http/assets/js/function.js
/srv/http/bash/cmd.sh cachebust
```

**Comment by Sagnasty** [comment]
Confirmed - works as normal now after applying commands.
